common/lib/common-definitions/src/test/validate0.20.0.ts

```diff
@@ -0,0 +1,331 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
it would probably be good to add a message that this file is autogenerated at some point